views/partials/footer.pug

```diff
@@ -0,0 +1,23 @@
+#footer
+
+    
+  //- CSS Style sheet for footer
```

Comment:
Let's move the style for this page to `public/stylesheets/style.css` You can create an id for the footer (`#` denotes an id, `.` denotes a class) The difference between them is an id can be used once on an object, but multiple classes can be used on an object. Since the footer is only used once (and therefore doesn't need variations) we can make it an id.

I'd create a large section denoted by a block comment like you've seen in the file and call it footer styles. Then you can call your style `#footer` or something similar.